docker-compose.yml

```diff
@@ -11,15 +11,27 @@ services:
     ports:
       - "5050:5050"
     depends_on:
-      - "mongo"
-  mongo:
+      - "graphelier-db"
```

Comment:
How does this work with volumes?

Reply:
> Can we rename `core/graphelier_news` to `core/graphelier-news` to match `core/graphelier-service`?
Well since graphelier_news is a python module i wasn't sure if naming it graphelier-news would work

Reply:
Since this module won't be imported directly I don't think that should be a problem, I don't have a strong preference on the exact name I would just like to have the naming conventions be consistent with existing directories.

Reply:
it will be for linting.

`CMD [ "pylint", "graphelier_news" ]`

Reply:
Tested it out locally with a package called `test-dash`, added an `__init__.py` and ran `pylint test-dash` without issues.

Reply:
The only problem would arise if we need to import something from outside the `graphelier-news` directory, say from the `scripts`. Then the solution would be to use string imports with `graphelier_news = __import__("graphelier-news")`

Reply:
As for the volume, it would use a new one